jax/_src/interpreters/pxla.py

```diff
@@ -3067,6 +3067,8 @@ def lower_sharding_computation(
   arg_names = None if arg_info is None else [
       f'{name}{keystr(path)}' for i, (name, path) in enumerate(arg_info)
       if i in kept_var_idx]
+  result_info = jaxpr.debug_info and pe.result_info(jaxpr.debug_info)
```

Comment:
Can you add this below too to lower_mesh_computation?

Reply:
Done!

Reply:
Thank you!